### PR TITLE
Move video call message state and duration to head.

### PR DESCRIPTION
**Pull request by aforge**
'webrtc' header for state.
'webrtc-duration' header for duration.

Content is now Drafty.videoCall() / VC.

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -6,3 +6,6 @@ export const CALL_STATE_IN_PROGRESS = 3;
 
 // Mime type used for video call messages.
 export const CALL_MESSAGE_MIME_TYPE = 'application/x-tinode-webrtc';
+
```
Comment (or-else) [inline src/constants.js:9]: `export const CALL_MESSAGE_MIME_TYPE = 'application/x-tinode-webrtc'; `

Remove?

File: src/widgets/chat-message.jsx [hunk 2]
```diff
@@ -120,7 +126,7 @@ class BaseChatMessage extends React.PureComponent {
 
     let content = this.props.content;
     const attachments = [];
-    if (this.props.mimeType == Drafty.getContentType() && Drafty.isValid(content)) {
+    if ([Drafty.getContentType(), CALL_MESSAGE_MIME_TYPE].includes(this.props.mimeType) && Drafty.isValid(content)) {
```
Comment (or-else) [inline src/widgets/chat-message.jsx:129]: `CALL_MESSAGE_MIME_TYPE `

Is it needed?

**Review by or-else — APPROVED** [review]

File: src/constants.js (hunk 1 above)
Comment (aforge) [reply src/constants.js]: Done.

File: src/widgets/chat-message.jsx (hunk 2 above)
Comment (aforge) [reply src/widgets/chat-message.jsx]: Removed.

**Pull request merged**